app/Widgets/Weather.php

```diff
@@ -22,10 +20,13 @@ public function run()
         /** @var \App\Services\AirportService $airportSvc */
         $airportSvc = app(AirportService::class);
         $metar = $airportSvc->getMetar($this->config['icao']);
+        $taf = $airportSvc->getTaf($this->config['icao']);
```

Comment:
Changed there. There shouldn't be a direct call to AviationWeather. That's breaking the abstraction layer - since I'm allowing it to be implemented differently if AviationWeather ever goes down

Reply:
Understood , thanks for the update